src/stores/wallet.ts

```
@@ -0,0 +1,193 @@
+    setOfflineApp(isOfflineApp) {
+      state.isOfflineApp = isOfflineApp
+    },
+    setLedgerApp(ledgerApp) {
+      state.ledgerApp = ledgerApp
+    },
+    setWallet(wallet) {
+      state.instance = wallet
+      state.address = wallet.getAddressString()
+      state.isHardware = wallet.isHardware ?? false
+      state.identifier = wallet.identifier
+      if (!wallet.hasOwnProperty('isHardware')) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)